src/de/cuuky/varo/configuration/configurations/config/ConfigSetting.java

```diff
@@ -20,9 +21,11 @@ public enum ConfigSetting implements SectionEntry {
 	// AUTOSETUP
 	AUTOSETUP_ENABLED(ConfigSettingSection.AUTOSETUP, "enabled", false, "Wenn Autosetup aktiviert ist, werden beim\nStart des Servers alle Spawns automatisch gesetzt und\noptional ein Autostart eingerichtet."),
 	AUTOSETUP_LOBBY_ENABLED(ConfigSettingSection.AUTOSETUP, "lobby.enabled", true, "Ob eine Lobby beim AutoSetup gespawnt werden soll"),
-	AUTOSETUP_LOBBY_HEIGHT(ConfigSettingSection.AUTOSETUP, "lobby.height", 10, "Hoehe der Lobby, die gespawnt werden soll"),
+	AUTOSETUP_LOBBY_SNAP(ConfigSettingSection.AUTOSETUP, "lobby.snap", AutoSetup.LobbySnap.MAX_HEIGHT.name(), "Naeherungsweise von welcher Position auf der Y-Achse\ndie Lobby gespawnt werden soll\n\nMoegliche Werte: GROUND, MAX_HEIGHT, ABSOLUTE"),
+	AUTOSETUP_LOBBY_SNAP_OFFSET(ConfigSettingSection.AUTOSETUP, "lobby.snapOffset", 50, "Wie weit von ihrem Snap-Punkt entfernt\ndie Lobby gespawnt werden soll"),
 	AUTOSETUP_LOBBY_SCHEMATIC(ConfigSettingSection.AUTOSETUP, "lobby.schematic", "plugins/Varo/schematics/lobby.schematic", "Schreibe hier den Pfad deiner Lobby-Schematic\nhin, die gepastet werden soll.\nHinweis: WorldEdit benoetigt"),
-	AUTOSETUP_LOBBY_SIZE(ConfigSettingSection.AUTOSETUP, "lobby.size", 25, "Groesse der Lobby, die gespawnt werden soll"),
+	AUTOSETUP_LOBBY_GEN_HEIGHT(ConfigSettingSection.AUTOSETUP, "lobby.gen.height", 10, "Wand-Hoehe der Lobby, die generiert werden soll"),
+	AUTOSETUP_LOBBY_GEN_SIZE(ConfigSettingSection.AUTOSETUP, "lobby.gen.size", 25, "Groesse der Lobby, die generiert werden soll"),

```

Comment:
I don't think the path of these should be changed as it breaks existing configs.

Reply:
> I understand the point, however the name (and the path) is very easy to get wrong. I think that this clarity about the function of the entry is worth it if these entries are reset in some configs

But I don't think `.gen` really does anything to clarify what these entries do. I'd suggest something like `lobby.size.xz`/`lobby.size.y` or `lobby.dimensions.width`/`lobby.dimensions.height`

> Especially since I would guess that very few users have actually changed this entry.

That's probably true

Reply:
> > I understand the point, however the name (and the path) is very easy to get wrong. I think that this clarity about the function of the entry is worth it if these entries are reset in some configs
> 
> But I don't think `.gen` really does anything to clarify what these entries do. I'd suggest something like `lobby.size.xz`/`lobby.size.y` or `lobby.dimensions.width`/`lobby.dimensions.height`

Both suggestions can still be misinterpreted as the general size of the lobby. But this setting only applies to the generated lobby (not the schematic lobby).
Maybe a mix of both, like `lobby.generated.dimensions.width` / `height`, would be the ideal solution.

Reply:
But why would one even want to try to change the size of a pre-made schematic in the config? If you really want to change the path of config entries, I would also change `lobby.schematic` to `lobby.schematic.path` (or maybe `lobby.schematic.file`) and add `lobby.schematic.enabled`.

Reply:
> But why would one even want to try to change the size of a pre-made schematic in the config?

That's a fair point, but I still think it should be made clear.

> If you really want to change the path of config entries, I would also change `lobby.schematic` to `lobby.schematic.path` (or maybe `lobby.schematic.file`) and add `lobby.schematic.enabled`.

With this proposal, there would then be the following entries:

```
lobby.schematic.enabled
lobby.schematic.file
lobby.generated.width
lobby.generated.height
```

If you don't mind I would implement it that way.